arp-speaker/Makefile

```diff
@@ -0,0 +1,61 @@
+SYSTEM:=
```

Comment:
I have to admit I'm really not a fan of the makefile format. I know it's super portable and available anywhere, but ugh it's such write-only code :(. That's why I used Fabric in the end, python is at least semi-maintainable.

But as I said in #47, if we can get all the stuff currently in fabfile into a makefile... I guess we could maybe switch to makefile. I'll hate it, but if it makes metallb easier to hack on...